Expanded Functionality

jfgreco opened the pull request:
- Added the THF
- Implemented get-divisions
- Separated the results by league
- Updated the readmes
- Added/Updated the actions
- Generalized functions to reduce code copy

jfgreco committed:
Update update.yml

jfgreco committed:
Automated ranking update: Thu Mar  2 00:57:00 UTC 2023

jfgreco committed:
Create thf.py

jfgreco committed:
Create krach.common.py

jfgreco committed:
Delete krach.common.py

jfgreco committed:
Create common.func.py

jfgreco committed:
030223-0708

jfgreco committed:
030223-0830

jfgreco committed:
Update readme.md

jfgreco committed:
Update readme.md

jfgreco committed:
Update readme.md

jfgreco committed:
030223-0906

jfgreco committed:
Expanded Functionality

Expanded the functionality of get_divisions and incorporated in the AHF and newly created THF routines.

jfgreco committed:
Update readme.md

jfgreco committed:
AHF Automated ranking update: Fri Mar  3 01:33:45 UTC 2023

jfgreco committed:
Added AGHF

jfgreco committed:
Update readme.md

jfgreco committed:
Merge branch 'main' of https://github.com/jfgreco/krach-ahf

jfgreco committed:
Update readme.md

jfgreco committed:
Update readme.md

jfgreco committed:
030423 - Update

-Consolidated repeated code
-Made generalized functions to use across multiple leagues

jfgreco committed:
AHF Automated ranking update: Sat Mar  4 17:38:22 UTC 2023

jfgreco committed:
Cleanup

jfgreco committed:
Output File Fix & Run of Results

jfgreco committed:
Create readme.md

jfgreco committed:
Create readme.md

jfgreco committed:
Create readme.md

jfgreco committed:
Create readme.md

jfgreco committed:
Merge branch 'main' of https://github.com/jfgreco/krach-ahf

jfgreco committed:
Merge branch 'main' of https://github.com/jfgreco/krach-ahf

jfgreco committed:
AHF Automated ranking update: Sat Mar  4 17:58:34 UTC 2023

jfgreco committed:
AGHF Automated ranking update: Sat Mar  4 23:17:07 UTC 2023

jfgreco committed:
Fix division by zero

Fixed division by zero error in the updateratings and showratings where there are no teams in a specific division.

jfgreco committed:
Merge branch 'main' of https://github.com/jfgreco/krach-ahf

jfgreco committed:
Fix mislabel in the workflow name

jfgreco committed:
Fixed THF Divisions

Added logic to ignore divisions that KRACH is not calculated for.

jfgreco committed:
THF Automated ranking update: Sun Mar  5 01:27:06 UTC 2023

jfgreco committed:
Fixed readme.md formating

jfgreco committed:
Merge branch 'main' of https://github.com/jfgreco/krach-ahf

jfgreco committed:
Organized common code

jfgreco committed:
AGHF Automated ranking update: Wed Mar  8 00:01:17 UTC 2023

jfgreco committed:
AHF Automated ranking update: Wed Mar  8 00:02:05 UTC 2023

jfgreco committed:
THF Automated ranking update: Wed Mar  8 00:02:40 UTC 2023

jfgreco committed:
Add result folder cleanup

jfgreco committed:
Fix delete command

jasonfsmitty commented:
Hey John, seems like you've been quite busy!  I'll be honest, I haven't reviewed any of this, but happy to do so once you're "done" (though our season is still going, so might be a while till I have the free time).

Regarding something I said in #7:
> Note that even with the changes, there are still some minor differences ... most teams are identical to the AHF published numbers, but there are still teams with different ratings. Any tweaks I've tried to make have ripple changes through all teams, making the overall ratings more different.

I still can't explain these differences.  I'm curious what you're seeing with your changes, whether the numbers exactly match the published KRACH ratings, or if you see something similar where a team here and there has a slightly different KRACH rating?

I did reach out to the AHF about the discrepancies, but I haven't heard back.

jfgreco committed:
Updated delete and download

The download command will now clean out the results directory before downloading new scores to ensure we aren't leaving any files around.

jfgreco committed:
Steamlined cleanup

jfgreco committed:
Updated workflow actions

jfgreco committed:
AGHF Automated ranking update: Fri Mar 10 12:03:52 UTC 2023

jfgreco committed:
AHF Automated ranking update: Fri Mar 10 12:04:41 UTC 2023

jfgreco committed:
THF Automated ranking update: Fri Mar 10 12:05:12 UTC 2023

jfgreco committed:
Fixed minor library name reference

jfgreco committed:
Update blackbear_common.py

Updated the SeasonId's for 2023-24 season.

jfgreco committed:
090423 Update

Fix a few bugs

jfgreco committed:
AHF Automated ranking update: Mon Sep  4 20:08:52 UTC 2023

jfgreco committed:
AHF Automated ranking update: Tue Sep  5 19:14:39 UTC 2023
